lib/read.js

```diff
@@ -128,3 +134,28 @@ module.exports = function wrapReadPixels (
 
   return readPixels
 }
+
+function isFloatSupported (gl) {
+  var texture = gl.createTexture()
+  gl.bindTexture(gl.TEXTURE_2D, texture)
+  gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA, 1, 1, 0, gl.RGBA, gl.FLOAT, null)
+
+  var fbo = gl.createFramebuffer()
+  gl.bindFramebuffer(gl.FRAMEBUFFER, fbo)
+  gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, texture, 0)
+  gl.bindTexture(gl.TEXTURE_2D, null)
+  if (gl.checkFramebufferStatus(gl.FRAMEBUFFER) !== gl.FRAMEBUFFER_COMPLETE) return false
+
+  gl.viewport(0, 0, 1, 1)
+  gl.clearColor(1.0, 0.0, 0.0, 1.0)
```

Comment:
I wonder if that can lose previous clear color, if there is any. That should not be an issue, since `regl.clear` or `regl._refresh` fully reinit the state.

Reply:
This check is run on startup, right? I think it'd be fine then as long as the state is refreshed/initialized afterwards.